### PR TITLE
chore: file menu upgrade (DHIS2-9821)

### PR DESCRIPTION
**Pull request by turban**
Fixes: https://jira.dhis2.org/browse/DHIS2-9821

This PR upgrades the file menu to the latest version. It required some code changes as we didn't use 'access', 'update', 'manage' and 'delete' fields for analytical objects. 

The PR also adds a missing translation. 

Before this PR: 

Screenshot 2021-02-16 at 10 11 15

After this PR: 

Screenshot 2021-02-16 at 10 10 44

### CODE REVIEW

**Comment by turban** [comment]
We get an error when using "Rename" and "Delete" in the File menu, so this PR can't be merged yet:

Screenshot 2021-02-16 at 11 38 18

File menu uses useDataMutation from @dhis2/app-runtime, and I presume we get this error because DHIS2 Maps is not on the app plattform yet #410

@amcgee is it possible to add DataProvider from app-runtime to get this working, or is it better/safer to wait for #410?

**Comment by turban** [comment]
Upgraded `@dhis2/app-runtime` to ^2.7.0

ran `npx yarn-deduplicate --packages @dhis2/app-runtime` to make sure the same version is used,

**Comment by turban** [comment]
Upgraded `@dhis2/ui` to ^6.2.2

**Comment by turban** [comment]
The above issue was solved by upgrading to `@dhis2/analytics` 15.0.0. 
It also required `styled-jsx` to be installed, which is now a peer dependency of `@dhis2/analytics`.

**Review by edoardo — APPROVED** [review]

File: src/components/app/FileMenu.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import D2FileMenu from '@dhis2/d2-ui-file-menu';
+import { FileMenu as D2FileMenu } from '@dhis2/analytics';
```
[the remark this answers is not in the document]
Comment (turban) [reply src/components/app/FileMenu.js:4]: Changed.

**Pull request merged**